### ISSUE TITLE
Missed OpenPilot Revolution support

### ISSUE THREAD

**Issue by night-ghost** [iNavFlight/inav#217]
There is a wonderful small controller Revo Mini, which supported by LibrePilot, recently even Ardupilot (https://github.com/ArduPilot/ardupilot/pull/3874) - but does not iNav.  There are Stm32f4 forks have been made, so why is it not in the list of supported cards?
In addition, this support will allow to compare the behavior of the same copter with different software

**Comment by digitalentity** [comment]
Blocked by #130

**Comment by night-ghost** [comment]
there is full REVO support - https://github.com/rs2k/raceflight

**Comment by digitalentity** [comment]
@night-ghost Who's going to port the code to INAV?

**Comment by martinbudden** [comment]
@digitalentity , I've been following the PR to add F4 support to betaflight, see PR https://github.com/borisbstyle/betaflight/pull/452 .

I think if @ledvinap 's PR https://github.com/cleanflight/cleanflight/pull/1421 "IO and EXTI driver" is merged into iNav the the F4 port is relatively straightforward - I'd certainly be willing to have a go at it.

**Comment by digitalentity** [comment]
@martinbudden you have a go :smile: 
I won't be able to crack the code for abput 2 more weeks so any help would be very much appreciated!

**Comment by martinbudden** [comment]
@ledvinap , would you have time to create an iNav PR for  https://github.com/cleanflight/cleanflight/pull/1421 "IO and EXTI driver" ?

One that is merged in I can start looking at doing the F4 work for iNav.
